crates/solidity/testing/sourcify/src/run.rs

```diff
@@ -92,7 +92,7 @@ fn run_parser_check(contract: &Contract, unit: &CompilationUnit, events: &Events
     let mut test_outcome = TestOutcome::Passed;
     for file in unit.files() {
         if !file.errors().is_empty() {
-            if let Ok(source) = contract.read_file(file.id()) {
+            if let Ok(Some(source)) = contract.read_file(file.id()) {
```

Comment:
I wonder if this would silently fail if the resolver has a bug?
Should we `unwrap()` any errors here?

Reply:
+4 other callsites below..

Reply:
not sure why was this resolved?